updates to riui for desktop launch (#4523)
GoogleChrome/developer.chrome.com: site/en/blog/richer-pwa-installation/index.md

To enable developers to provide installed experiences on par with native experiences
Chrome is introducing a new install surface, Richer Install, that allows developers to add a
description and screenshots to their manifest file and have it appear in a bottomsheet dialog
within Chrome for Android.

<figure>

>   {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/RoQRxu3CPkKvAtkxYsAQ.jpeg", alt="Example of bottomsheet UI in Chrome", width="360", height="720" %}

Reviewer: ```suggestion
  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/RoQRxu3CPkKvAtkxYsAQ.jpeg", alt="Example of bottomsheet UI in Chrome.", width="360", height="720" %}
```

This gives developers the opportunity to create a more enticing install process
that better aligns to user expectations and that mimics their existing mental model
of installed experiences.

{% Columns %}
{% Column %}
<figure>
  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/SpStAtUk8Zp5iwi9yqKP.jpg",

> alt="Richer Install UI Expanded", width="342", height="722" %}

Reviewer: ```suggestion
alt="Richer Install UI Expanded.", width="342", height="722" %}
```

  <figcaption>Richer Install UI Expanded.</figcaption>
</figure>
{% endColumn %}
{% Column %}
<figure>
  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/k7r4yKqrh6iOm2XyZHfw.jpg",

> alt="Richer Install UI Collapsed", width="342", height="722" %}

Reviewer: ```suggestion
alt="Richer Install UI Collapsed.", width="342", height="722" %}
```

## Backwards compatibility {: #compatibility }

Websites that do not include at least one screenshot to their manifest file will continue to receive
the existing prompts. This may change in the future depending on uptake of the developer community
and users' reaction.

## Previewing the UI {: #previewing }

> This UI works in M94 on Android, and [M108 on desktop](/blog/richer-install-ui-desktop/).

Reviewer: ```suggestion
This UI works from Chrome 94 on Android, and [Chrome 108 on desktop](/blog/richer-install-ui-desktop/).
```

This feature is enabled on [squoosh.app](https://squoosh.app) and can be previewed there.

## Implementation {: #implementation }

> To display the richer install ui dialog developers need to add at least one screenshot for the corresponding form factor in the `screenshots` array. The `description` field is not required but it is recommended,  the content dialog is built with the content of those two fields and make the experience more similar to app store install, helping users identify they are adding an app to their device and with more space available developers can provide specific context to their users at install time.

Reviewer: ```suggestion
To display the richer install UI dialog developers need to add at least one screenshot for the corresponding form factor in the `screenshots` array. The `description` field is not required but it is recommended. The content dialog is built using the content of the `screenshots` and `description` fields to make the experience more similar to an app store install. This UI helps users identify they are adding an app to their device, and with more space available developers can provide specific context to their users at install time.
```

## Previewing the UI {: #previewing }

> This UI works in M94 on Android, and M108 on desktop.

Reviewer 2: Shall we link to your upcoming article for the `M108 on desktop` part?
